app/controllers/api/functions.php

```diff
@@ -956,9 +956,23 @@
             /** Update execution status */
             $execution->setAttribute('status', $executionResponse['status']);
             $execution->setAttribute('statusCode', $executionResponse['statusCode']);
-            $execution->setAttribute('stdout', $executionResponse['stdout']);
-            $execution->setAttribute('stderr', $executionResponse['stderr']);
             $execution->setAttribute('time', $executionResponse['time']);
+
+            $stdoutMax = 0;
+            $stderrMax = 0;
+            $attributes = Config::getParam('collections')['executions']['attributes'];
+            foreach ($attributes as $attribute) {
+                switch($attribute['$id']) {
+                    case 'stdout':
+                        $stdoutMax = $attribute['size'];
+                        break;
+                    case 'stderr':
+                        $stderrMax = $attribute['size'];
+                        break;
+                }
+            }
+            $execution->setAttribute('stderr', \substr($executionResponse['stderr'], 0, $stderrMax));
```

Comment:
We should use `mb_substr ` instead of `substr ` to be able to support unicode characters as well 
https://stackoverflow.com/a/56238294/7659504